h5io/_h5io.py

```diff
@@ -193,6 +188,10 @@ def _triage_write(key, value, root, comp_kw, where,
                       where + '.csr_matrix_shape', cleanup_data=cleanup_data,
                       slash=slash)
     else:
+        try:
+            from pandas import DataFrame, Series
+        except ImportError:
+            DataFrame = Series = type(None)
```

Comment:
This should just be `pass`, and the following 7 lines should be in the `else` clause of the `try` block